# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/all.js
# Commit: ca2e6339f35f
# Full Hash: ca2e6339f35f38cc26a203f29c8dbfeb38f96afc
# Author: Kelsey Gilbert <kelsey.gilbert@mozilla.com>
# Date: 2023-03-14 09:41:39
# Regressor Bug: 1799258
# File Overlap Count: 1
# Description:
#   Bug 1799258 - Do color-management on Windows+DComp via IDCompositionFilterEffects. r=sotaro
#   
#   + Add gfx.color_management.rec709_gamma_as_srgb:true. :'(
#   
#   In particular, rec709(16/255) -> srgb(31/255). Even though it's
# ==============================================================================

diff -r aa4e010199c6 -r ca2e6339f35f modules/libpref/init/all.js
--- a/modules/libpref/init/all.js	Mon Mar 13 21:04:10 2023 +0000
+++ b/modules/libpref/init/all.js	Mon Mar 13 21:04:10 2023 +0000
@@ -449,8 +449,6 @@
   pref("gfx.hidpi.enabled", 2);
 #endif
 
-pref("gfx.color_management.display_profile", "");
-
 pref("gfx.downloadable_fonts.enabled", true);
 pref("gfx.downloadable_fonts.fallback_delay", 3000);
 pref("gfx.downloadable_fonts.fallback_delay_short", 100);
